lib/Agrammon/UI/CommandLine.pm6

```diff
@@ -95,6 +97,44 @@ multi sub MAIN('run', Str $filename, ExistingFileOrStdin $input, ExistingFile :$
     say $output;
 }
 
+#| Get inputs from dataset
+multi sub MAIN(
+        'getDataset', Str $model-filename, Str $username, Str $dataset-name,
+        Str $version, Str $gui, Str $variant, ExistingFile :$cfg-file
+    ) is export {
+    my %data := get-dataset $model-filename, $username, $dataset-name, $version, $gui, $variant, :$cfg-file;
+    my @output;
+    @output.push("## User: %data<username>");
+    @output.push("## Dataset: %data<dataset>");
+    if %data<validation-errors> {
+        note "WARNING: validation errors";
```

Comment:
Why do we `note` this rather than putting it into the output?

Reply:
> Why do we `note` this rather than putting it into the output?

Because I expect the user to redirect the output to a file and possibly overlook the fact that are validation errors.
The next lines are writing the actual errors to  the output.

Reply:
Ah, OK, that makes sense.